static/styles/subscriptions.css

```diff
@@ -1070,17 +1070,11 @@
     }
 }
 
-#change_stream_info_modal {
-    .modal-header h3 {
-        text-overflow: ellipsis;
-        overflow: hidden;
-    }
-
-    #change_stream_description {
-        width: 80%;
-        max-width: 500px;
-        box-sizing: border-box;
-    }
+#change_stream_description {
+    width: 100%;
+    height: 80px;
+    margin-bottom: 4px;
+    box-sizing: border-box;
```

Comment:
Any reasons for changing this height and width of this input? I think if we want to change this, then would be better in a separate commit.

Reply:
The original ones looked odd after the migration, so I changed them.

Reply:
Yeah, these do look better.  I verified that things don't look broken with an extremely long stream name, though it's possible the previous overflow settings were better.